### PR TITLE
Lets aloe cream autorepeat when healing

### PR DESCRIPTION
**Pull request by googobbug**
## About The Pull Request

Allows aloe cream to automatically repeat on application. Also minor grammar stuff.

## Why It's Good For The Game

~~In #50558, the stack size of sutures was reduced from 15 down to 10. Not only does this appear to be a mistake (not referenced anywhere in the changelog, appears to be a holdover from an early version of the PR where all sutures on station were replaced with the botany sutures), but it made the already poor healing density of sutures even worse. I don't think changing the stack size back to 15 is unreasonable, especially given that they're now important for stopping bleeding.~~

As for the aloe cream, it currently sucks to actually heal anyone with it due to the amount of clicking required. Making it autorepeat like every other healing item solves this issue.

## Changelog
:cl: Bumtickley00
fix: Medicated sutures no longer yield more polypyr than it takes to make them when ground
tweak: Aloe cream now automatically repeats when healing
spellcheck: Aloe cream is no longer referred to as "medical stack" in the description
/:cl:

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
can you speak to the grinding change so i dont get another "this pr didnt document this feature therefore it must be a mistake".

**Comment by AdmiralPancakes1** [comment]
Changelog has been updated to mention that.

**Comment by ExcessiveUseOfCobblestone** [comment]
@Ryll-Ryll

**Comment by Ryll-Ryll** [comment]
Yeah whatever I guess 15 seems like a lot but whatever

**Comment by Rohesie** [comment]
> 
> 
> Yeah whatever I guess 15 seems like a lot but whatever

What number would be saner?

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -195,8 +195,8 @@
 	icon_state = "suture"
 	self_delay = 30
 	other_delay = 10
-	amount = 10
-	max_amount = 10
+	amount = 15
+	max_amount = 15
```
Comment (Rohesie) [inline code/game/objects/items/stacks/medical.dm:199]:
```suggestion
	amount = 10
	max_amount = 10
```
Remove this buff and it goes in

**Pull request merged**